scripts/migrations/30-compiled-contracts-column.py

```diff
@@ -0,0 +1,23 @@
+"""
```

Comment:
@olonho this is not how the database migration should be done. This was an old way of doing protocol upgrades which we have abandoned. For how we do database migrations, please take a look at https://github.com/near/nearcore/blob/802671e7412af975924c98d77c7f464906fe1f27/neard/src/lib.rs#L66